### DISCUSSION TITLE
Rectangle Pro and LanguageTool Desktop

### DISCUSSION THREAD

**Discussion by johanneswiedmann** [rxhanson/RectanglePro-Community#409] [Q&A] [answered]
When having Rectangle Pro and LanguageTool Desktop (https://languagetool.org/mac-desktop) running at the same time, window snapping by drag-and-drop with the mouse doesn't work anymore. The snap areas don't appear in that case.

Once LanguageTool Desktop is closed, window snapping works again right away.

Using Rectangle Pro shortcuts on the keyboard is not affected by this issue.

Any idea how I could work around this?

**Comment by rxhanson** [comment, marked as answer]
Please try executing this terminal command and let me know if it fixes the issue for you:

`defaults write com.knollsoft.Hookshot systemWideMouseDown -int 1`

Then restart Rectangle Pro.

If that doesn't fix it, then it might have to be fixed in LanguageTool.

**Comment by johanneswiedmann** [comment]
Works like a charm! Thanks a lot for the help!
